src/bin/migrate.rs

```diff
@@ -418,6 +418,11 @@ fn migrations() -> Vec<Migration> {
                              crate_owners_unique_user_per_crate", &[]));
             Ok(())
         }),
+        Migration::new(20150320174400, |tx| {
+            try!(tx.execute("ALTER TABLE keyword SET keyword = lower(keyword)",
+                            &[]));
```

Comment:
I think that this should be `ALTER TABLE keywords` (although I also thought it would be `UPDATE keywords ...`?).

I think that this may also not land well in production as there's currently a uniqueness constraint on the `keyword` column, but this will generate duplicates (e.g. FFT => fft).

I think that this may want to iterate through all keywords, deleting those with a downcase equivalent and migrating all current members of that keyword to the downcase equivalent. This should also probably delete the old uniqueness constraint and add a new one on `lower(keyword)` to speed up queries and provide a db-level gurantee that we don't have duplicates.